TAC/Meetings/2023/2023-08-10/CCC-TAC-Minutes-2023-08-10.md

```diff
@@ -0,0 +1,132 @@
+* DM welcomed the members of the TAC and reviewed the values of the CCC and the antitrust policy of the Linux Foundation
+* Kurt Taylor (KT) assisted with the meeting minutes
+* DM reviewed the agenda
+
+### Roll Call / Attendance
+
+***Voting members of the TAC***
+
+*Per the [charter](https://charter.confidentialcomputing.io), all [CCC Premier members](https://confidentialcomputing.io/members/) receive one vote on the TAC. Quorum for votes is at least 50% of voting members present.*
+
+* Accenture - Giuseppe Giordano 
+* Arm - Thomas Fossati / Michael Lu
+* Google - Cfir Cohen + / Catalin Sandu 
+* Huawei - Zhipeng (Howard) Huang 
+* Intel - Dan Middleton (TAC chair) + / Simon Johnson
+* Meta - Eric Northup / Shankaran Gnanashanmugam
+* Microsoft - Dave Thaler +
+* Red Hat - Lily Sturmann / Yash Mankad +
+
+   " + " indicates attending
+
+***Other Attendees***
+
+* Thore Sommer (Keylime/FHNW)
+* Jorg Rodel (SUSE)
+* Ofir Azoulay-Rozanes (Anjuna)
+* Mark Novak (JP Morgan Chase)
+* Georg Link (CHAOSS)
+* Leonardo Garcia (Linaro)
+* Shalom Shefa (Cisco)
+* Eric Voit (Cisco)
+* Simon Frost (Arm)
+* Sung Lee (VMware)
+* Hani Benhabiles (ByteDance)
+* Pawan Khandavilli (Microsoft)
+* Ryan Gifford (CSA)
+* Olu Oniyinde (Intel)
+* Alec Fernandez (Microsoft)
+* Drasko Draskovic (Ultraviolet)
+* Ye Li (VMware)
+* Mike Ferron-Jones (Intel)
+* Jiewen Yao (Intel)
+* Hernan Gatta (Opaque)
+* Helen Lau (LF)
+* Mike Bursell (CCC)
+* Kurt Taylor (LF)
+
+
+**Agenda Addition**
+
+* MB discussed the Mini Summit and sponsorships at OSS EU, seeking papers for the schedule
+
+
+**Meeting Minute Approval**
+
+Proposed: That the minutes of the July 13, 2023 meeting of the Technical Advisory Council of the Confidential Computing Consortium as distributed to the members of the TAC in advance of this meeting are hereby adopted and approved.
+
+* DM reviewed the prior meeting minutes
+* Quorum was reached and the minutes were approved by the voting members of the TAC
+
+
+**GRC SIG Update**
+
+* Mark Novak (MN) presented the SIG update
+* MN reviewed the GRC goals
+* Joint CCC/[CSA](https://cloudsecurityalliance.org/research/working-groups/confidential-computing/) working group, updates to the CSA Cloud Control Matrix (CSM) v4
+* Ryan Gifford discussed the work on the CCM in the CSA, [Attestation Governance Patterns](https://docs.google.com/document/d/1QV0baEDrbd4HK4dgHuk10BRs8552L2HePVpxYWSqLXw/edit#heading=h.pu882iozwrwx), RATS & Governance
+* Discussion ensued
+* Need help getting involved with NIST, if anyone has contacts it would be helpful
+
+
+**Rust SPDM**
+
+* Jiewen Yao presented an introduction to SPDM and proposal for new project
+* [Security Protocol and Data Model (SPDM)](https://www.dmtf.org/standards/spdm)
+* Component Measurement and Authentication (CMA)
+* [Rust-SPDM project](https://github.com/intel/rust-spdm) - Rust implementation of the SPDM protocol
+* There were questions about the project being a good fit in the CCC, discussion ensued
```

Comment:
```suggestion
* There were questions about whether the project is a good fit in the CCC, since the project is not specific to confidential computing.  Discussion ensued
```